### ISSUE TITLE
Replace Chrome logo

### ISSUE THREAD

**Issue by tydog98** [standardnotes/web#262]
The button to launch the webapp should have a more neutral icon. Chrome is not the entire web.

**Comment by mobitar** [comment]
Honestly couldn't find a better icon to represent browser. We use Font Awesome. If you have any suggestions please let me know.

**Comment by tydog98** [comment]
[It appears Font Awesome has a generic browser icon](https://fontawesome.com/icons/browser?style=regular)

**Comment by bryvin** [comment]
@tydog98 You're not wrong there but that is part of the Pro bundle which Font Awesome does charge for.. albeit not too much but I don't believe the current version is using the Pro icons.

**Comment by tydog98** [comment]
If Standard Notes doesn't have access to the Pro version, perhaps this [log-in icon](https://fontawesome.com/icons/sign-in-alt?style=solid) could work? It would make sense as clicking the button brings you right to the log-in page.

**Comment by bryvin** [comment]
Possibly the external-link icons.. there are 2 free variations: https://fontawesome.com/icons?d=gallery&q=external

**Comment by mobitar** [comment]
I'll keep this in mind next time I'm in there making changes.

**Comment by jstavanja** [comment]
How about the [globe icon](https://fontawesome.com/icons/globe?style=solid)? It is featured in the majority of the results that pop up when I Google search "_font awesome icon web_".
